go/vt/vttablet/tabletmanager/vreplication/replicator_plan.go

```diff
@@ -394,15 +406,33 @@ func (tp *TablePlan) applyChange(rowChange *binlogdatapb.RowChange, executor fun
 		if tp.isOutsidePKRange(bindvars, before, after, "insert") {
 			return nil, nil
 		}
-		return execParsedQuery(tp.Insert, bindvars, executor)
+		if !tp.isPartial(rowChange) {
```

Comment:
take it or leave it: personal preference to ask a positive question rather than a negative question. ie., ask `if tp.isPartial(...)`, and flip the order of if-else execution blocks.

Reply:
Agreed, fixed